tcms/testruns/tests/test_views.py

```diff
@@ -444,15 +445,75 @@ def setUpTestData(cls):
         cls.case_run_1.case.text = "Scenario Version 2"
         cls.case_run_1.case.save()
 
-    def test_update_selected_case_runs(self):
+        cls.update_case_run_text_html = \
+            '<a href="#" title="Update the IDLE case runs to newest case text" ' \
+            'href="javascript:void(0)" data-param="{0}" ' \
+            'class="updateBlue9 js-update-case">{1}</a>' \
+            .format(
+                reverse('testruns-update_case_run_text', args=[cls.test_run.pk]),
+                _('Update')
+            )
```

Comment:
Instead of this rather ugly HTML snippet you can modify the template instead. Add a unique id attribute for this menu option and you can assert for presence/non-presence of the id value instead of the text + all of the other HTML attributes.